### PR TITLE
🌐 Add Hungarian translation for `/docs/hu/docs/index.md`

### PR DESCRIPTION
**Pull request by takacs**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 40bb4a040ff03ec9a96000275ef0a1f4c957987d at: https://f134e758.fastapitiangolo.pages.dev

**Review by pdavid345 — APPROVED** [review]
Native Hungarian, approving translation.

**Comment by tiangolo** [comment]
📝 Docs preview for commit c5cedd2cea579c90194ccf1973826a14af4206e0 at: https://77cc6bab.fastapitiangolo.pages.dev

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:38]: Typeo?

Comment (panko) [inline docs/hu/docs/index.md:38]:
```suggestion
* **Egyszerű**: Egyszerű tanulásra és használatra tervezve. Kevesebb dokumentáció olvasással töltött idő.
```

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:41]:
```suggestion
* **Szabvány alapú**: Az API-ok nyílt szabványaira alapuló (és azokkal teljesen kompatibilis): <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (korábban Swagger néven ismert) és a <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
```
maybe like this?

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:64]: Do we want to translate the reviews? Hard to decide.

Comment (takacs) [reply docs/hu/docs/index.md:64]: I looked at other translations and most of them didn't have the reviews translated.

Comment (panko) [reply docs/hu/docs/index.md:64]: then it's decided, thanks

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:199]: Maybe typeo here?

Comment (takacs) [reply docs/hu/docs/index.md]: Fixed!

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:260]: I don't have special cases right now here, but maybe "la'sd" instead of "provided by"?

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:328]: "deklaralod"?

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:379]:
```suggestion
* Validája hogy az `item_id` `int` típusú a `GET` és `PUT` kérésekben.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 6537dfde7d884383f7774c10784194963a621cf6 at: https://14049bfc.fastapitiangolo.pages.dev

**Comment by takacs** [comment]
@panko I added all of your suggested changes.

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:397]: Here maybe we can figure out something clever. "Eddig csak a felszint kapargattuk, de a lenyeg hogy most mar konyebben ertheto hogyan mukodik." something like this?

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md]:
```suggestion
...erre:
```

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:417]:
```suggestion
... És figyeld meg hogy a szerkesztő automatikusan tudni fogja a típusokat és kiegészíti azokat:
```

File: docs/hu/docs/index.md
Comment (takacs) [reply docs/hu/docs/index.md]: I noticed this one when adding your previous suggestions it should be fixed.

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:421]:
```suggestion
Teljesebb példákért és funkciókért tekintsd meg a <a href="https://fastapi.tiangolo.com/tutorial/">Tutorial - User Guide</a> -t.
```

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:423]:
```suggestion
**Spoiler veszély**: a Tutorial - User Guidehoz tartozik:
```
maybe we could use the "izelito" word here?
something like 
"**Spoiler veszély**: izelito a Tutorial - User Guide -bol:"?

File: docs/hu/docs/index.md
Comment (panko) [inline docs/hu/docs/index.md:440]: Could we reword this a bit please?

**Comment by panko** [comment]
LGTM, thanks for the hungarian translation!

File: docs/hu/docs/index.md
Comment (takacs) [reply docs/hu/docs/index.md:440]: I couldn't find better wording for this. This is the original:

_Independent TechEmpower benchmarks show FastAPI applications running under Uvicorn as [one of the fastest Python frameworks available](https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7), only below Starlette and Uvicorn themselves (used internally by FastAPI). (*)_

**Comment by tiangolo** [comment]
📝 Docs preview for commit e9740824eb002751be654bbecdb2b2bea38ba51b at: https://7db5951e.fastapitiangolo.pages.dev

File: docs/hu/docs/index.md
Comment (panko) [reply docs/hu/docs/index.md:440]:
```A független TechEmpower benchmarkok szerint az Uvicorn alatt futó **FastAPI** alkalmazások az <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">egyik leggyorsabb Python keretrendszerek közé tartoznak</a>, éppen lemaradva a Starlette és az Uvicorn (melyeket a FastAPI belsőleg használ) mögött.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit c266f8417512adefd6fe819ca8c184a91c09eada at: https://b647c251.fastapitiangolo.pages.dev

File: docs/hu/docs/index.md
Comment (takacs) [reply docs/hu/docs/index.md]: Changed!

**Review by panko — APPROVED** [review]
LGTM

**Comment by takacs** [comment]
hey @tiangolo got both approvals, is there anything else to do before merging this PR?

**Comment by tiangolo** [comment]
Great, thank you @takacs! ☕ 

And thanks for the reviews @panko, @pdavid345! 🍰

**Pull request merged**
